packages/neutrino/src/api.js

```diff
@@ -18,7 +18,7 @@ class Api {
 
   // getOptions :: Object? -> IO Object
   getOptions(opts = {}) {
-    let moduleExtensions = new Set(['js', 'jsx', 'vue', 'ts', 'mjs', 'json']);
+    let moduleExtensions = new Set(['js', 'jsx', 'vue', 'ts', 'tsx', 'mjs', 'json']);
```

Comment:
What is `tsx`? TypeScript with JSX?

Reply:
Yep. https://www.typescriptlang.org/docs/handbook/jsx.html